### PR TITLE
Fix some errors in the README

### PR DESCRIPTION
**Pull request by bjorn3**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

Fix some errors in the README.

### CODE REVIEW

**Review by fu5ha — APPROVED** [review]

**Comment by bjorn3** [comment]
@fu5ha is this waiting on something before it can be merged?

**Comment by fu5ha** [comment]
Nope, I'm used to automerge bots at work and forgot x) Thanks for the ping

**Pull request merged**
